Frontend/CO.CDP.OrganisationApp/Pages/ClaimOrganisationInvite.cshtml.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.AspNetCore.Mvc;
+using ProblemDetails = CO.CDP.Person.WebApiClient.ProblemDetails;
+using CO.CDP.OrganisationApp.Constants;
+using CO.CDP.Person.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Pages;
+
+public class ClaimOrganisationInviteModel(
+    IPersonClient personClient,
+    ISession session) : LoggedInUserAwareModel(session)
```

Comment:
Use PageModel base class unless need to fetch UserDetail

Reply:
@dharmverma I use UserDetail on line 14 to get the UserUrn: 
<img width="863" alt="image" src="https://github.com/user-attachments/assets/c8aef0c6-7e3d-46b3-9756-d1e642ba0054">

Reply:
Warnings have gone. I switched to using SessionContext